src/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM debian:unstable-20200607
+LABEL maintainer="richard.steinbrueck@googlemail.com"
+
+## Configuration ##############################################################
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN ln -snf /usr/share/zoneinfo/UTC /etc/localtime && echo UTC > /etc/timezone
+
+## Init #######################################################################
+COPY src/supervisord/supervisord-sshd.conf /etc/supervisor/conf.d/sshd.conf
+COPY src/supervisord/supervisord-nginx.conf /etc/supervisor/conf.d/nginx.conf
+COPY src/supervisord/supervisord.conf /etc/
+
+# Runtime config ##############################################################
+COPY src/entrypoint.sh /entrypoint.sh
+COPY src/entrypoint.d /dev/entrypoint.d
+RUN chmod 755 /entrypoint.sh
+
+# Packages ####################################################################
+COPY src/packages.txt /tmp/packages.txt
+# we cleanup later and i use cat because i like cat!
+# hadolint ignore=DL3009,SC2002
+RUN apt-get update && cat /tmp/packages.txt | xargs apt-get -q -y -o Dpkg::Use-Pty=0 install
```

Comment:
```suggestion
# I use cat because i like cat!
# hadolint ignore=DL3009,SC2002
RUN apt-get update && apt-get -q -y -o Dpkg::Use-Pty=0 install $(cat /tmp/packages.txt) && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

This can reduce the image size a bit:
<img width="705" alt="image" src="https://user-images.githubusercontent.com/15176026/85922539-6e10a580-b884-11ea-9e1f-7b438226e762.png">